### PR TITLE
SWARM-1609: Upgrading the Teiid Fraction

### PR DESCRIPTION
**Pull request by rareddy**
- [ x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x ] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [ x] Have you built the project locally prior to submission with `mvn clean install`?

-----

### CODE REVIEW

File: build-parent/pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
     <version.wildfly.config-api>1.1.0.Final</version.wildfly.config-api>
 
     <version.keycloak.config-api>1.1.0.Final</version.keycloak.config-api>
-    <version.teiid.config-api>1.0.2</version.teiid.config-api>
+    <version.teiid.config-api>1.0.3-SNAPSHOT</version.teiid.config-api>
```
Comment (kenfinnigan) [inline build-parent/pom.xml]: This needs to be a released version, not a SNAPSHOT

File: fractions/teiid/components/accumulo/pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (kenfinnigan) [inline fractions/teiid/components/accumulo/pom.xml]: Formatting in pom.xml files needs to be fixed to match the project

File: fractions/teiid/pom.xml [hunk 3]
```diff
@@ -1,151 +1,51 @@
...
+		<groupId>org.wildfly.swarm</groupId>
+		<artifactId>build-parent</artifactId>
+		<version>2017.11.0-SNAPSHOT</version>
+		<relativePath>../../build-parent/pom.xml</relativePath>
+	</parent>
+
+	<name>Teiid Fraction</name>
+	<description>Data Virtualization based on Teiid</description>
+
+	<artifactId>teiid-fraction</artifactId>
```
Comment (kenfinnigan) [inline fractions/teiid/pom.xml]: This should be `teiid-parent`

File: fractions/teiid/pom.xml [hunk 4]
```diff
@@ -1,151 +1,51 @@
...
+	<packaging>pom</packaging>
+
+	<modules>
+		<module>core</module>
+		<module>components</module>
+		<module>odata</module>
+	</modules>
+
+	<!-- Profiles -->
+	<profiles>
```
Comment (kenfinnigan) [inline fractions/teiid/pom.xml]: Why is this profile needed here?

File: fractions/teiid/components/pom.xml [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.wildfly.swarm</groupId>
+		<artifactId>teiid-fraction</artifactId>
+		<version>2017.11.0-SNAPSHOT</version>
+	</parent>
+
+	<name>Teiid Components</name>
+	<artifactId>components</artifactId>
```
Comment (kenfinnigan) [inline fractions/teiid/components/pom.xml]: Artifact id should be `teiid-components`. `components` is too generic

**Comment by kenfinnigan** [comment]
Could you also rebase on the current master, as there's a conflict

**Comment by rareddy** [comment]
@kenfinnigan Code rebased and pom.xml file formatted.

**Comment by kenfinnigan** [comment]
ok to test

**Comment by rareddy** [comment]
"teiid-components" based fractions are not showing up in the "bom-all" pom file, thus I can't seem to include them in the project, do know what I could be doing wrong?

**Comment by rareddy** [comment]
@kenfinnigan Now I got it correctly done, can you please review.

**Review by kenfinnigan — APPROVED** [review]

**Pull request merged**